posixlib/src/main/scala/scala/scalanative/posix/netdb.scala

```diff
@@ -50,6 +50,8 @@ object netdb {
 
   @name("scalanative_AI_CANONNAME")
```

Comment:
Just a comment since this C code already exists. I think it is much more pleasing to use all lowercase even if the C macro is all uppercase. I think we should search through on a another PR to normalize these. @LeeTibbert, do you concur?

Reply:
@ekrich  Yes, I agree,  The existing line (& siblings) should not be changed for this PR and
that either you or I should create an Issue about the casing of `@name(""scalanative_foo`).
I just checked posixlib and posix/sys and the usage appears to be systematically random between files.

In fact, just the other day I created a PR adding support for posix/sys/resource. I used scananative_FOO
because the models I happened to pick out of the directory all used that form and I wanted to be
consistent with them.  I think the scalanative_foo form has greater consistency with the larger scope of the project.

Reply:
I think that would look nicer also, but agree that it doesn't belong in this PR